user/templates/movie_list.html

```diff
@@ -8,14 +8,18 @@
     <h1>Movie List</h1>
     <ul>
         {% for movie in movies %}
-            <div style="padding: 10px;">
-                <li>{{ movie.title }}</li>
-                <li>{{ movie.description }}</li>
-                <li><img src="{{ movie.poster.url }}" width="100"></li>
-            </div>
-        {% empty %}
-            <li>No movies available.</li>
-        {% endfor %}
+    <div style="padding: 10px;">
+        <li>{{ movie.title }}</li>
+        <li>{{ movie.description }}</li>
+        {% if movie.poster %}
```

Comment:
movie poster must always be there so remove this if condition

Reply:
if we are entering all the data by ourself  and doesn't give the poster then it handles the error

Reply:
make the posters a required field

Reply:
Yes it should be like that

Reply:
i am only taking title from the frontend for api fetch so how can i make poster required at client side whereas in server side if the data comming from api then it will have poster     if i make poster required then i need to make two forms one only contains title and other that contain all the fields where i can make all fields required

Reply:
Ok interesting. One thing you can do is fetch the details of the movie only from the api. So if a weird movie title like Eleena is entered we should'nt save that in the database. Or what you can do is if the image isn't entered save a default image as the poster.

Reply:
it already has the feature that if title not found and do not entered any other field then it discard the entry   so i just need to remove the other fields from frontend?

Reply:
yes it can be removed